test/tests/functional/pbs_verify_log_output.py

```diff
@@ -113,31 +112,30 @@ def test_if_info_add(self):
         """
         Test for interface info presence in log files
         """
-        names = "["
-        for name in [x[0] for x in self.all_interfaces()]:
-            names += "(" + name + ")"
-        names += "]"
+        interfaceGenerator = self.all_interfaces()
+        for ifname in interfaceGenerator:
+            name = "[(" + ifname + ")]"
 
-        log_val = "[(ipv4)(ipv6)(ipv4/ipv6)] [(interface: )]" + names
+            log_val = "[( interface: )]" + name
```

Comment:
I don't know enough about how [] works in python with ()s.  It's meant to choose one of many (the book says one character of many).  If we assume that things inside of ()s are matched as a whole just like a character would, I can understand that.  Why only have one set between [] though?  You have [(interface: )].  Wouldn't that be the same thing as if you dropped the []s?